GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle more contributions, from a larger and more diverse group of contributors.
+
+## The R package
+
+The purpose of the project is to maintain the R data.table package,
+which is guided by the following principles:
+
+* Few (if any) dependencies
+* Time & memory efficiency
+* Concise syntax (minimal redundancy in code)
+* Stable code base (easy for users to upgrade to new data.table, and compatible with old R versions)
```

Comment:
```suggestion
* Stable code base (strong preference for back-compatibility with data.table itself and with old versions of R)
```

possibly

Reply:
I liked "easy for users" because of how it makes you think of the user experience when upgrading, but I like "back-compatible" too

Reply:
How about "strong preference for user-friendly back-compatiblity..." ? Agree user-centricity is important to emphasize.